RobotCode/MecanumDrive/src/org/usfirst/frc4915/MecanumDrive/subsystems/Elevator.java

```diff
@@ -146,8 +142,30 @@ public void setPosition(double position) {
      * 
      * @param positionNumber
      */
-	public void convertPositionToHeight(int positionNumber) {
-		// TODO Auto-generated method stub
-		
+	public double convertPositionToHeight(int positionNumber) {
+		switch (positionNumber) {
+		case 0:
+			positionNumber = POSITION_ZERO;
+			break;
+		case 1:
+			positionNumber = POSITION_ONE;
+			break;
+		case 2:
+			positionNumber = POSITION_TWO;
+			break;
+		case 3:
+			positionNumber = POSITION_THREE;
+			break;
+		case 4:
+			positionNumber = POSITION_FOUR;
+			break;
+		case 5:
+			positionNumber = POSITION_FIVE;
+			break;
+		case 6:
+			positionNumber = POSITION_SIX;
+			break;
```

Comment:
I'd add a `default:` case to this - `positionNumber` isn't guaranteed to be `0 <= x <= 6`